go.mod

```diff
@@ -351,6 +354,7 @@ require (
 
 replace (
 	github.com/Azure/azure-sdk-for-go => github.com/elastic/azure-sdk-for-go v59.0.0-elastic-1+incompatible
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/consumption/armconsumption => github.com/gpop63/azure-sdk-for-go/sdk/resourcemanager/consumption/armconsumption v0.0.0-20221201195319-8eb84523c8e3
```

Comment:
@gpop63, as you already mentioned over Slack, we should use github.com/elastic/azure-sdk-for-go. 

We can probably find the right version and create a branch and a tag.